java/src/main/java/org/rocksdb/WriteBatch.java

```diff
@@ -106,6 +106,12 @@ public Handler() {
     public abstract void delete(byte[] key);
     public abstract void logData(byte[] blob);
 
+    public void singleDelete(byte[] key) {}
+    public void singleDelete(int columnFamilyID, byte[] key) {}
```

Comment:
Would we be better to pass a `ColumnFamilyReference` here?

Reply:
I think in terms of maintaining consistency with the Java API, we should try and use a `ColumnFamilyHandle`. I will see if I can find a mechanism for resolving one from the column_family_id.

Reply:
@min-mwei I found some a mechanism for going from column family id to column family hanle, see here: https://github.com/facebook/rocksdb/pull/1298#pullrequestreview-68691985

Reply:
Hi, @adamretter, just saw this as for some reason the email notifications went to the junk folder. Please feel free to take over the changes, as I am not active on this at the moment